content/docs/command-reference/add.md

```diff
@@ -243,3 +246,44 @@ In this case, a DVC-file is generated for each file in the `pics/` directory
 tree. No top-level DVC-file is generated, which is typically less convenient.
 For example, we cannot use the directory structure as one unit with `dvc run` or
 other commands.
+
+## Example: Dvcignore
+
+Let's take an example to illustrate how `.dvcignore` interacts with `dvc add`.
+
+```dvc
+$ mkdir dir
+$ echo file_one > dir/file1
+$ echo file_two > dir/file2
+```
+
+Now add `file1` to `.dvcignore` and track the entire `Dir` directory with
+`dvc add`.
+
+```dvc
+$ echo Dir/file1 > .dvcignore
+$ dvc add Dir
+```
+
+Let's now modify `file1`(which is listed in `.dvcignore`) and run `dvc status`.
+
+```dvc
+$ echo file_one_changed > dir/file1
+$ dvc status
+Data and pipelines are up to date.
+```
+
+`dvc status` ignores changes to files listed in `.dvcignore`. Let's have a look
+at cache directory.
+
+```dvc
+$ tree .dvc/cache
+.dvc/cache
+├── 0a
+│   └── ec3a687bd65c3e6a13e3cf20f3a6b2.dir
+└── 52
+    └── 4bcc8502a70ac49bf441db350eafc2
+```
+
+Only checksums of directory(`Dir/`) and `file2` have been cached. See
```

Comment:
"the checksums"
But actually let's use "the hash values" please.

> I'm also changing that term from the Dvcignore guide here: 3f2d554b

Reply:
Also, missing another space in there...

Reply:
Also, the 2 sentences are not related either. Please separate them even if each one is a separate paragraph, that's OK. Thanks